### ISSUE TITLE
Can we generate texture after finishing the mesh simplification?

### ISSUE THREAD

**Issue by lcsdz** [cdcseacave/openMVS#314]
### Steps to reproduce the problem

Thank you very much for the system you provided.
One of the questions for you is about texture mapping. Some simplified mesh methods are used to simplify the mesh model generated by RefineMesh. Can we use the next TextureMesh to generate textures? This simplification method can be Quadric edge collapse decimation and so on.

I use the command of TextureMesh `scene.mvs --mesh-file <mesh.ply>`, Where mesh.ply has been simplified using the above method,and run to `Assigning the best view to each face completed: 33469`, this step will automatically exit, is this OK? What should I do?

**Comment by lcsdz** [comment]
Thank you very much for your reply. My problem has been resolved.
In addition, I would like to ask another question about system configuration.
I reconfigured the MVS system and compiled it all.But when the ReconstructMesh is executed, it crashes.
The log shows such information：

> 22:44:03 [App     ] OS: Windows 10+ x64
22:44:03 [App     ] SSE & AVX compatible CPU & OS detected
22:44:03 [App     ] Command line: scene_dense.mvs -w E:\MvgMvsPipeline\output\mvs
22:44:03 [App     ] error: invalid stream (unsupported version)

The program prompts an error and displays 981 lines of error in the SceneReconstruct.cpp
`ASSERT(facets.empty() && inter.type == intersection_t::VERTEX && inter.v1 == vi);`

What I want to ask is, is it because my CGAL library is too new? I'm using this version of 4.11.1.Or what else?Look forward to your reply again.
